src/services/commands.js

```diff
@@ -0,0 +1,153 @@
+Controller.open(function(_) {
+
+  var COMMAND_CONFIGURATION_TYPE_MAP = {
+    symbol: 'vanillaSymbol',
+    variable: 'variable',
+    nonSymbola: 'nonSymbolaSymbol',
+    variable: 'binary'
+  }
+
+  _.initializeLatexGrammar = function() {
+    this.latexCmds = Object.assign({}, LatexCmds);
+    this.charCmds = Object.assign({}, CharCmds);
+    this.textCommands = {}
+
+    this.cursor.options.autoCommands = {}
+    
+    // Initialize the grammar processors for various symbols
+    // by loading up the default configuration 
+    // This is WIP. Eventually all files symbol definitions
+    // should be transferred over to the defaultSymbolDefinitions file
+    this.extendLatexGrammar(GLOBALLY_DISABLED_INPUT, 'symbol');
+    // this.extendLatexGrammar(VANILLA_SYMBOLS, "symbol");
```

Comment:
Does this need to be completed as part of this release?  If not we will need to track it with a Jira issue.